bin/check-marathon-task.rb

```diff
@@ -173,4 +170,13 @@ def check_tasks(data)
 
     [tasks.length, unhealthy]
   end
+
+  def default_tasks(server)
+    expected_tasks_url = "/v2/apps/#{config[:task]}"
```

Comment:
might be worth exposing the api version as an option.

Reply:
That could make sense but if the API version has breaking changes we'll need to change the check code so I don't see much gain, usually when a new API version is made available the old one will be kept at least for a few versions so, I think it's better/safer to support one specific version of the API.